packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -65,23 +65,15 @@ MeetingsUtil.checkForCorrelationId = (deviceUrl, locus) => {
 };
 
 MeetingsUtil.parseUserPreferences = (userPreferences) => {
```

Comment:
the method name is "parseUserPreferences" but it returns jus a single thing - the siteUrl, so either we should either:
1. rename it to say "parseSiteUrlFromUserPreferences" or
2. change it to return an object that contains the parsed information and for now it might have just 1 field, that's OK, it will probably grow in the future when we start using other fields.

personally I would prefer the 2nd option

Reply:
Thanks for the comment @marcin-bazyl, Let me check it and update you.

Reply:
Renaming it as `parseSiteUrlFromUserPreferences` this util is specific to the Webex Sites.